src/jstests/formio/components/datetime.spec.js

```diff
@@ -0,0 +1,227 @@
+import set from 'lodash/set';
+import {Formio} from 'react-formio';
+
+import MinMaxDatetimeValidator from 'formio/validators/minMaxDatetimeValidator';
+
+const FormioComponent = Formio.Components.components.component;
+
+describe('Datetime Component', () => {
+  test('Datetime validator: no min/max datetime', done => {
+    const component = {
+      label: 'datetime',
+      key: 'datetime',
+      type: 'datetime',
+      datePicker: {
+        minDate: null,
+        maxDate: null,
+      },
+      customOptions: {
+        allowInvalidPreload: true,
+      },
+      validate: {datetimeMinMax: true},
+    };
+
+    const componentInstance = new FormioComponent(component, {}, {});
+
+    const isValid = MinMaxDatetimeValidator.check(
+      componentInstance,
+      {},
+      '2020-01-01T10:00:00+01:00'
+    );
+
+    expect(isValid).toBeTruthy();
+
+    done();
+  });
```

Comment:
this is all synchronous code, no? so we can leave out the `done` argument + callback I think?